src/io/flutter/run/OpenDevToolsAction.java

```diff
@@ -34,7 +35,11 @@ public OpenDevToolsAction() {
   }
 
   public OpenDevToolsAction(@NotNull final FlutterApp app, @NotNull final Computable<Boolean> isApplicable) {
-    super("Open DevTools", "Open Dart DevTools", FlutterIcons.Dart_16);
+    super(
+      "Open DevTools",
```

Comment:
can we make this text the same?
` "Open " + (FlutterModuleUtils.declaresFlutter(app.getProject()) ? "Flutter" : "Dart") + " DevTools",`

Reply:
This is the Flutter plugin so I don't think you can actually run an app unless it declares Flutter.